app/controllers/comments_controller.rb

```diff
@@ -8,7 +8,7 @@ def new
   def create
     @post = Post.find(params[:post_id])
     @user = @post.user
-    @comment = @post.comments.new(user_id: @user.id, text: commnet_params[:text])
+    @comment = @post.comments.new(user_id: @post.id, text: comment_params[:text])
```

Comment:
`user_id:` should take the `current_user` id, not the `@post` id, Kindly fix it.

Reply:
please I have tried it and the form is no longer submit, any idea?